tests/attr/test_deconvolution.py

```diff
@@ -10,8 +10,11 @@
     NeuronDeconvolution,
 )
 
+from typing import Any, Tuple, Union, List
+from torch.nn import Module
 from .helpers.basic_models import BasicModel_ConvNet_One_Conv
 from .helpers.utils import assertTensorAlmostEqual, BaseTest
+from captum.attr._utils.typing import TensorOrTupleOfTensors
 
 
 class Test(BaseTest):
```

Comment:
The test methods below need to be type hinted as well, most of them will just have return -> None.

Reply:
done